src/components/Post/CommentDialogBox.jsx

```diff
@@ -29,86 +25,78 @@ const CommentDialogBox = ({
   deleteCommentID,
 }) => {
   return (
-    <Box sx={{ flexGrow: 1 }}>
-      <Grid container spacing={2}>
-        <Grid item xs={6} md={6}>
-          <Item>
-            {postHasImages ? (
-              <ImageSlider slides={postImages} isCommentBox />
-            ) : (
-              <div className="post__background">
-                <p className="post_caption">{caption}</p>
-              </div>
-            )}
-          </Item>
-        </Grid>
-        <Grid item xs={6} md={6}>
-          <Scroll>
-            <Item>
-              <div className="post__comments">
-                {comments.length ? (
-                  <>
-                    {comments.map((userComment) => (
-                      <p key={userComment.id}>
-                        <strong>{userComment.content.username}</strong>{" "}
-                        {userComment.content.text}
-                        <span
-                          onClick={() => {
-                            setOpenToDeleteComment(!openToDeleteComment);
-                            setDeleteCommentID(userComment);
-                          }}
+    <Box sx={{
+      overflowY: "scroll", '&::-webkit-scrollbar': {
+        width: 0,
+      }
+    }} borderRadius="16px" maxHeight="30vh">
+      {comments.length ? (
+        <>
+          {comments.map((userComment) => (
+            <div key={userComment.id}>
+              <div style={{ padding: "0.4rem 1rem", marginTop: "0.3rem", marginBottom: "0.3rem", display: "flex", justifyContent: "space-between", background: "lightgray", borderRadius: "10px" }}>
```

Comment:
can we use a css class to store these many styles, it's not right to use inline styles